OPHD/UI/IconGrid.h

```diff
@@ -26,18 +26,14 @@ class IconGrid : public Control
 	/**
 	 * Item used within the IconGrid.
 	 */
-	class Item
+	struct Item
 	{
 	public:
```

Comment:
A `struct` defaults to `public` access, so the `public:` access specifier can be removed with this change.

Reply:
Wherps, forgot to remove that. That's why I changed it to a struct over a class.